libraries/botbuilder-ai/botbuilder/ai/qna/models/qna_request_context.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from msrest.serialization import Model
+
+
+class QnARequestContext(Model):
+    """ The context associated with QnA.  Used to mark if the current prompt is relevant with a previous question or not. """
```

Comment:
pylint error, since the line is over 120 characters long

Please break the comment down into multiple lines

Reply:
Done